src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1235,21 +1240,25 @@ begin
   { intro h,
     split,
     { intros v w,
-      obtain ⟨p, hp⟩ := h v w,
+      obtain ⟨p, hp⟩ := h.2 v w,
       use p, },
-    { rintros v w ⟨p, hp⟩ ⟨q, hq⟩,
-      simp only,
+    { cases h with hmpty h,
+      split,
+      swap,
+      simp [hmpty],
+      rintros v w ⟨p, hp⟩ ⟨q, hq⟩,
+      simp only [p, hp, q, hq],
```

Comment:
Non-final `simp`s are prohibited by our coding style. If this is a final `simp`, then it should be in `{ simp }` to make it clear. Also, with the suggested changes above you can revert this chunk.
```suggestion
    { rintros v w ⟨p, hp⟩ ⟨q, hq⟩,
      simp only [p, hp, q, hq],
```